mlflow/server/js/src/components/ExperimentViewUtil.js

```diff
@@ -134,6 +140,7 @@ export default class ExperimentViewUtil {
       getHeaderCell("run_name", <span>{"Run Name"}</span>),
       getHeaderCell("source", <span>{"Source"}</span>),
       getHeaderCell("source_version", <span>{"Version"}</span>),
+      getHeaderCell("tags", <span>Tags</span>),
```

Comment:
We should probably disable the click-to-sort toggle on this header cell - i.e. it's currently possible to click (presumably to sort) on "Tags", but this doesn't seem to actually sort on anything:

![image](https://user-images.githubusercontent.com/2358483/59173116-38c1da00-8b00-11e9-8847-79e975640160.png)

Reply:
Disabled sorting for this column.